parity-genesis.template.json

```diff
@@ -46,9 +46,17 @@
-        "0000000000000000000000000000000deadbeef1": { "balance": "1000000000000000000000" },
-        "0000000000000000000000000000000deadbeef2": { "balance": "1000000000000000000000" },
-        "0000000000000000000000000000000deadbeef3": { "balance": "1000000000000000000000" },
-        "0000000000000000000000000000000deadbeef4": { "balance": "1000000000000000000000" }
+        "000000000000000000000000000000deadbeef10": { "balance": "1000000000000000000000" },
```

Comment:
Why start at deadbeef10?

Reply:
Because to be a valid hex it needs to be a certain length. Either deadbeef10 or we start at deadbeef1 and at 10 we drop a zero from the prefix.